### DISCUSSION TITLE
I can't add a folder to a document

### DISCUSSION THREAD

**Discussion by renevanputten** [gingerbeardman/stapler#23] [Q&A] [answered]
I tried to add a folder to a stapler document but I can only add individual items

**Comment by gingerbeardman** [comment, marked as answer]
This is currently possible using drag and drop.

I had to disable it via the file selector because there were issues, I may take a look at it again in future.

<img width="569" alt="Screen shot 2024-08-19 at 22 50 51" src="https://github.com/user-attachments/assets/8d0502e0-d2ea-4839-b139-a2f270056790">

**Comment by renevanputten** [comment]
Thanks I will try this

Regards,
Rene van Putten
